Makefile

```diff
@@ -342,6 +347,7 @@ ADDONS_BINDING_SOURCES := \
 	$(filter-out test/addons/??_*/*.cc, $(wildcard test/addons/*/*.cc)) \
 	$(filter-out test/addons/??_*/*.h, $(wildcard test/addons/*/*.h))
 
+.PHONY: test/addons/.buildstamp
```

Comment:
If we're going to make this target `.PHONY`, I would remove the `touch` so it's no longer a file and probably rename the target so it doesn't look like a file either.